### DISCUSSION TITLE
Ckan auto load with plugins

### DISCUSSION THREAD

**Discussion by cascaoSDC** [ckan/ckan#6094] [Q&A]
i'm new to ckan, and i would like to know how can i make it so that the ckan is already loaded with plugins that i whant .... i tried to install using the dockerfile file and also tried to replace the production.ini file but nothing worked would it be possible to advise me on this matter?

`
[...]
# Install FIWARE specific extensions
RUN pip install -e git+https://github.com/conwetlab/ckanext-datarequests.git#egg=ckanext-datarequests && \
    pip install -e git+https://github.com/conwetlab/ckanext-fiware_header.git#egg=ckanext-fiware_header && \
    pip install -e git+https://github.com/okfn/ckanext-fiwarelabs.git#egg=ckanext-fiwarelabs && \
    pip install -e git+https://github.com/ckan/ckanext-geoview.git#egg=ckanext-geoview && \
    pip install -e git+https://github.com/telefonicaid/ckanext-ngsiview#egg=ckanext-ngsiview && \
    pip install -e git+https://github.com/conwetlab/ckanext-oauth2#egg=ckanext-oauth2 && \
    pip install -e git+https://github.com/ckan/ckanext-pdfview.git#egg=ckanext-pdfview && \
    pip install -e git+https://github.com/ckan/ckanext-harvest.git#egg=ckanext-harvest && \
    pip install -e git+https://github.com/conwetlab/ckanext-privatedatasets.git#egg=ckanext-privatedatasets && \
    pip install -e git+https://github.com/conwetlab/ckanext-storepublisher.git#egg=ckanext-storepublisher
# cp pre-build production.ini
COPY "myconfig/production.ini.teste"  "/etc/ckan/production.ini"
[...]
`

**Comment by frafra** [comment]
CKAN extensions requires installing `requirements.txt` (or similar) in addition to using pip. See: https://github.com/ckan/ckan/discussions/6601
